plugins/generic/externalFeed/simplepie/SimplePie.inc.php

```diff
@@ -9188,7 +9188,7 @@ function get_element($realname, $string)
 						{
 							$attribs[$j][2] = $attribs[$j][1];
 						}
-						$return[$i]['attribs'][strtolower($attribs[$j][1])]['data'] = SimplePie_Misc::entities_decode(end($attribs[$j]), 'UTF-8');
+						$return[$i]['attribs'][strtolower($attribs[$j][1])]['data'] = SimplePie_Misc::entities_decode(end($attribs[$j]));
```

Comment:
This is 3rd-party code, and we're trying not to distribute modified versions if possible.